### PR TITLE
Feat 719 map dialog component

### LINKED ISSUE #719: [Feat] create map dialog
image
- create molecule MapDialog
- dialog should include
   - header 
   - main section
   - footer with buttons
   - section that display chosen section 
- make sure to add translation for : header, chosen section, buttons
* Note: NO need to add the map
#678 
waiting for #721

### PR DESCRIPTION
**Pull request by daniel-ilan**
closes #719

### CODE REVIEW

**Review by SejoB — APPROVED** [review]

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -105,5 +105,11 @@
       "compared to national average": "in a section compared to national average on roads of the same type"
     },
     "No data": "No data to display"
+  },
+  "mapDialog": {
+    "searchButton": "Search",
+    "cancelButton": "Cancel",
+    "chosenSegment": "Chosen section: ",
```
Comment (yuvalbl) [inline public/locales/en/translation.json:112]: why the space?

File: public/locales/he/translation.json [hunk 2]
```diff
@@ -105,5 +105,11 @@
       "compared to national average": "במקטע לעומת ממוצע ארצי בכבישים מאותו הסוג"
     },
     "No data": "אין נתונים להצגה"
+  },
+  "mapDialog": {
+    "searchButton": "חיפוש",
+    "cancelButton": "ביטול",
+    "chosenSegment": "המקטע שנבחר: ",
```
Comment (yuvalbl) [inline public/locales/he/translation.json:112]: same here
spaces can be done with padding or &nbsp;
should not be part of i18n files (as this is easy to break by mistake)

File: src/components/molecules/MapDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      <Box className={classes.wrapper}>
+        <DialogTitle className={classes.dialogHeader}>
+          <Typography.Title1>{t('mapDialog.searchSection')}</Typography.Title1>
+        </DialogTitle>
+        <DialogContent>
+          {/*A placeholder to visually estimate the size*/}
+          <img src="https://images.indianexpress.com/2017/05/google-maps-759.jpg" alt="" />
+          <div className={classes.chosenSection}>
+            {/*needs to be bolder - just give it a classname?*/}
+            <Typography.Body1 bold>{t('mapDialog.chosenSegment')}</Typography.Body1>
```
Comment (daniel-ilan) [inline src/components/molecules/MapDialog.tsx:52]: The text should be bolder per the design, I tried using the "bold" prop we have but it doesn't seem to work... did I miss something?  should I just give it a classname?

File: src/components/molecules/MapDialog.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  isShowing: boolean;
+  onClose: () => void;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    dialogFooter: {
+      display: 'flex',
+      justifyContent: 'flex-end',
+      gap: theme.spacing(1.5),
```
Comment (yuvalbl) [inline src/components/molecules/MapDialog.tsx:18]: please use round values

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**